### PR TITLE
SAAS-1211 migrate getting started to MUI

### PR DESCRIPTION
**Pull request by nicolalamacchia**
image
image
image

### CODE REVIEW

File: src/components/GettingStarted/GettingStarted.tsx [hunk 1]
```diff
@@ -10,7 +11,9 @@ export const GettingStarted: FC = () => {
 
   return (
     <>
-      <header className={styles.header}>{Messages.getStarted}</header>
+      <Typography variant="h5" className={styles.header}>
```
Comment (TomasBlazek3PG) [inline src/components/GettingStarted/GettingStarted.tsx:14]: SHould we use variant or component? I guess component?

File: src/components/GettingStarted/GettingStartedOrgSection/GettingStartedOrgSection.tsx [hunk 2]
```diff
@@ -18,7 +19,7 @@ export const GettingStartedOrgSection: FC = () => {
     <GettingStartedSection
       description={Messages.createOrganizationDescription}
       title={Messages.createOrganizationTitle}
-      linkIcon={hasOrg ? undefined : 'plus-circle'}
+      linkIcon={hasOrg ? undefined : <Add />}
```
Comment (TomasBlazek3PG) [inline src/components/GettingStarted/GettingStartedOrgSection/GettingStartedOrgSection.tsx:22]: maybe hasOrg && <Add />?

File: src/components/GettingStarted/GettingStartedSection/GettingStartedSection.styles.ts [hunk 3]
```diff
@@ -1,77 +1,72 @@
...
-    descriptionWrapper: css`
-      align-items: center;
-      display: flex;
-      margin-left: calc(${spacing.lg} + ${headerImageSize});
-    `,
-    header: css`
-      align-items: center;
-      display: flex;
-      font-size: ${typography.heading.h4};
+    > h5 {
```
Comment (TomasBlazek3PG) [inline src/components/GettingStarted/GettingStartedSection/GettingStartedSection.styles.ts:20]: Why we dont target it dirrectly?

File: src/components/GettingStarted/GettingStarted.tsx (hunk 1 above)
Comment (nicolalamacchia) [reply src/components/GettingStarted/GettingStarted.tsx:14]: we should use `variant` as per documentation

File: src/components/GettingStarted/GettingStartedOrgSection/GettingStartedOrgSection.tsx (hunk 2 above)
Comment (nicolalamacchia) [reply src/components/GettingStarted/GettingStartedOrgSection/GettingStartedOrgSection.tsx:22]: no, this is correct, `false && 'Test'` would result in `false`, we need undefined

File: src/components/GettingStarted/GettingStartedSection/GettingStartedSection.styles.ts (hunk 3 above)
Comment (nicolalamacchia) [reply src/components/GettingStarted/GettingStartedSection/GettingStartedSection.styles.ts:20]: fixed

**Review by TomasBlazek3PG — APPROVED** [review]

**Pull request merged**